src/css/profile/mobile/common/listview.less

```diff
@@ -205,6 +205,9 @@ tau-expandable {
 			overflow-x: hidden;
 			text-overflow: ellipsis;
 			white-space: nowrap;
+			font-size: 18 * @px_base;
+			color: @color_popup_listview_1line;
```

Comment:
try to use css variables instead of "less" variables.
eg. var(--popup-listview-1line)

Reply:
This is the same color like `var(--text-color)`

Reply:
@TomaszLukawskiSam 
**1. try to use css variables instead of "less" variables.
eg. var(--popup-listview-1line)**
-> I have replace @color_popup_listview_1line to var(--popup-listview-1line).
done.

**2. This is the same color like var(--text-color)**
-> I know what the same --text-color. 
However, the color is differently displayed in the specification document according to the lignt and dark theme. I think it is good to put it intentionally.